lib/rack/prx_auth/certificate.rb

```diff
@@ -11,6 +11,7 @@ class Certificate
 
       def initialize(cert_uri = nil)
         @cert_location = cert_uri.nil? ? DEFAULT_CERT_LOC : URI(cert_uri)
+        @certificate = nil
```

Comment:
This prevents a warning further down - trying to access `@certificate` when it's not initialized.